types/evidence.go

```diff
@@ -336,8 +336,8 @@ func RegisterMockEvidences(cdc *amino.Codec) {
 // DuplicateVoteEvidence contains evidence a validator signed two conflicting
 // votes.
 type DuplicateVoteEvidence struct {
-	VoteA *Vote
-	VoteB *Vote
+	VoteA *Vote `json:"vote_a"`
+	VoteB *Vote `json:"vote_b"`
```

Comment:
👍 Looks like we need also need to remove the pubkey field in the rust version too: https://github.com/informalsystems/tendermint-rs/pull/242/files#r417475910